app/styles/notification.css

```diff
@@ -1,8 +1,12 @@
+.notifications-holder {
+    display: flex;
```

Comment:
Why we even need flex? we have divs so anyhow they are going on new line.

Reply:
Flex is there to center the notifications